tests/lustre/syntax-errors/contract-error.lus

```diff
@@ -0,0 +1,12 @@
+contract bla_spec(in: real) returns (out: bool) ;
+let
+  guarantee out = (in > 0.0);
+tel
+
+node bli(in: real) returns (out: bool) ;
+(*@contract
+  import bla_spec (in) returns (out+out) ;
+*)
+let
+  out = in >= 0.0 ;
+tel
```

Comment:
Why is this test in the **syntax-error directory** if Kind 2 returns a type error?

Reply:
`import bla_spec (in) returns (out+out);`
This should be a syntax error... 
I get this error 
```
<Error> line 8 col. 36: Failure cause: Syntax Error! Perhaps an illegal expression in contract call.
```